Dockerfile

```diff
@@ -59,20 +60,23 @@ COPY --chown=axom:axom . /home/axom/workspace
 WORKDIR /home/axom/workspace
 RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=Debug -DENABLE_OPENMP=On -DCMAKE_CXX_FLAGS=-fsanitize=address ..
 RUN cd build && make -j 16
+RUN cd build && ./bin/raja-perf.exe --checkrun -sp
 
 FROM axom/compilers:nvcc-10.2 AS nvcc10
 ENV GTEST_COLOR=1
 COPY --chown=axom:axom . /home/axom/workspace
 WORKDIR /home/axom/workspace
 RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=g++ -DENABLE_CUDA=On -DCMAKE_CUDA_STANDARD=14 ..
 RUN cd build && make -j 2
+RUN cd build && ./bin/raja-perf.exe --dryrun
```

Comment:
I would only add the args for debug builds if the standard release build run (no args) completes in a reasonable amount of time. However, add the '-sp' arg to all run cases. This will print to the log file the kernel names as each one runs. If an issue arises, it's helpful to know which kernel failed. AFAIK, that's the only way to know this without running through a debugger.

Reply:
I'll add -sp to all cases. Previously we weren't even running raja-perf for device builds. I added --dryrun for device builds so we at least check to see if we can run the executable, doing any runs would throw a CUDAassert.

Reply:
Makes sense. Thanks.